### PR TITLE
upgraded guzzlehttp to 7.8

### PR DESCRIPTION
**Pull request by MaheshDharmappa**
Upgraded guzzlehttp from 6.3 to 7.8

### CODE REVIEW

File: src/FusionExport/ExportManager.php [hunk 1]
```diff
@@ -72,20 +77,26 @@ public function exportAsStream(ExportConfig $exportConfig, $exportBulk=true) {
 			$tempDir = tempnam('tmp', 'fusioncharts');
 			$fileName = $tempDir . '_export.zip';
 
+            
+
 			file_put_contents($fileName, $contents);
-			$zipFile = new \ZipArchive();
+			$zipFile = new ZipArchive;
```
Comment (meherhowji-5740) [inline src/FusionExport/ExportManager.php:83]: is this intentional?

Comment (meherhowji-5740) [reply src/FusionExport/ExportManager.php:83]: new ZipArchive without parenthesis

Comment (MaheshDharmappa) [reply src/FusionExport/ExportManager.php]: My Bad, not an intentionally. Added Parenthesise.

Comment (meherhowji-5740) [reply src/FusionExport/ExportManager.php]: Can you please run this functionality and check one please. Thanks.

**Comment by meherhowji-5740** [comment]
@MaheshDharmappa also please rebase and squash your commits.

**Comment by meherhowji-5740** [comment]
@MaheshDharmappa please rebase and squash the commits as well. thanks

File: src/FusionExport/ExportManager.php (hunk 1 above)
Comment (MaheshDharmappa) [reply src/FusionExport/ExportManager.php]: Hi
i checked, files are exporting and working fine.

**Comment by meherhowji-5740** [comment]
Also, please upgrade the version from 2.0.0 to 2.1.0

**Comment by MaheshDharmappa** [comment]
Updated the version from 2.0.0 to 2.1.0 in composer and squash the commits.

**Pull request merged**